src/constants.js

```diff
@@ -1,3 +1,8 @@
+import speedoPkg from '../package.json'
+import webdriverPkg from 'webdriver/package.json'
+
+export const SPEEDO_USER_AGENT = `webdriver/${webdriverPkg.version} (Speedo/${speedoPkg.version})`
```

Comment:
I would suggest to have two different user agents here. We use one for making a WebDriver session and the other for making an API call, how about:

- WebDriver UA: `webdriver/${webdriverPkg.version} (Speedo/${speedoPkg.version})`
- Speedo API call UA: `speedo/<SPEEDO_VERSION> (nodejs <PLATFORM>)`